automated-record-card/index.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="https://contentful.github.io/ui-extensions-sdk/cf-extension.css">
+    <script src="https://unpkg.com/contentful-ui-extensions-sdk@3"></script>
+    <style>
+      .cf-form-field {
+        height: 220px;
+        margin-bottom: 0;
+      }
+
+      #thumbnail-preview {
+        float: right;
+        max-height: 200px;
+        max-width: 35%;
+      }
+
+      #thumbnail-preview div {
+        height: 200px;
+        width: 100%;
+        background: #ddd;
+      }
+
+      #thumbnail-preview img {
+        width: auto;
+        max-height: 200px;
+      }
+
+      input.cf-form-input {
+        width: 60%;
+      }
+    </style>
+  </head>
+  <body>
+
+    <div class="cf-form-field">
+      <div id="thumbnail-preview">
+        <div></div>
+      </div>
+      <input type="text" placeholder="Paste your recordID" class="cf-form-input"/>
```

Comment:
Change the placeholder text to include "record ID" i.e. with a space between the two terms.